apps/crn-server/test/handlers/webhooks/cronjob-sync-orcid.test.ts

```diff
@@ -30,31 +44,31 @@ describe('Cronjob - Sync Users ORCID', () => {
     );
     patchedUser.data.orcidWorks.iv = fixtures.ORCIDWorksDeserialisedExpectation;
 
+    const userResponse = getGraphQLUser();
+    const response = generateGraphqlFetchUsersResponse([
+      {
+        ...userResponse,
+        flatData: {
+          ...userResponse.flatData,
+          orcidLastSyncDate: '2020-01-01T00:00:00.000Z',
+          orcid,
+        },
+      },
+    ]);
+
+    jest
+      .spyOn(SquidexGraphql.prototype, 'request')
+      .mockResolvedValueOnce(response)
+      .mockResolvedValueOnce(getSquidexUserGraphqlResponse())
+      .mockResolvedValueOnce(fetchUserResponse());
+
     nock(config.baseUrl)
-      .get(`/api/content/${config.appName}/users`)
-      .query({
-        q: JSON.stringify({
-          take: 30,
-          filter: {
-            path: 'data.orcid.iv',
-            op: 'contains',
-            value: '-',
-          },
-          sort: [
-            {
-              path: 'data.orcidLastSyncDate.iv',
-              order: 'ascending',
-            },
-          ],
-        }),
-      })
-      .reply(200, fixtures.fetchUsersResponse)
       .patch(
-        `/api/content/${config.appName}/users/57d80949-7a75-462d-b3b0-34173423c490`,
-        matches({
-          email: { iv: patchedUser.data.email.iv },
-          orcidWorks: { iv: fixtures.ORCIDWorksDeserialisedExpectation },
-        }),
+        `/api/content/${config.appName}/users/user-id-1`,
+        // matches({
```

Comment:
are you sure this should stay in? do we not want to test it?

Reply:
Done